server/src/test/java/io/druid/curator/announcement/AnnouncerTest.java

```diff
@@ -128,7 +128,7 @@ public void eventReceived(CuratorFramework client, CuratorEvent event) throws Ex
     Assert.assertNull("expect /somewhere/test2 unannounced", curator.checkExists().forPath(testPath2));
   }
 
-  @Test(timeout = 60_000L)
+  //@Test(timeout = 60_000L)
```

Comment:
`@Ignore` is better than commenting out.

Any idea why this test is sketchy?

Maybe link to the open issue in a comment here.